<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DirectInteractionWindow</class>
 <widget class="QWidget" name="DirectInteractionWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Direct Interaction Window</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ScannerInspectionTool.qrc">
    <normaloff>:/DirectInteractionWindow/Resources/icon-direct.png</normaloff>:/DirectInteractionWindow/Resources/icon-direct.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QSpinBox" name="apiSelection">
       <property name="contextMenuPolicy">
        <enum>Qt::NoContextMenu</enum>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>150</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="submitBtn">
       <property name="text">
        <string>Make Request</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Result</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="apiCode">
       <property name="text">
        <string>None</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="apiResponse">
     <property name="plainText">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Request Data First</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ScannerInspectionTool.qrc"/>
 </resources>
 <connections/>
</ui>
